README.md

```diff
@@ -392,7 +392,7 @@ instance = SingleVMOpenEdXInstance.objects.create(
     sub_domain='dogwood.sandbox',
     name='Dogwood',
     fork_name='edx/edx-platform',
-    branch_name='named-release/dogwood',
+    branch_name='named-release/dogwood.rc',
```

Comment:
@antoviaque @mtyaka I suggest we leave the README alone for now: With the changes from #63, `branch_name` is no longer a field belonging to instances; and since that PR is going to land very soon (the only thing that's left is for @bradenmacdonald to update the README :D), there's no need to change/work on the instructions for manual provisioning in this context. (Even if this PR lands before #63, we'd only have to live with slightly outdated instructions for another day, which seems manageable.)